packages/core/content-manager/admin/src/index.ts

```diff
@@ -44,6 +44,38 @@ export default {
     });
 
     app.registerPlugin(cm.config);
+
+    // Register homepage widgets
+    app.widgets.register([
+      {
+        icon: Pencil,
+        title: {
+          id: `${PLUGIN_ID}.widget.last-edited.title`,
+          defaultMessage: 'Last edited entries',
+        },
+        component: async () => {
+          const { LastEditedWidget } = await import('./components/Widgets');
+          return LastEditedWidget;
+        },
+        pluginId: PLUGIN_ID,
+        id: 'last-edited-entries',
+        permissions: [{ action: 'plugin::content-manager.explorer.read' }],
+      },
+      {
+        icon: CheckCircle,
+        title: {
+          id: `${PLUGIN_ID}.widget.last-published.title`,
```

Comment:
did you check if translation files need to be updated?

Reply:
The translations were already coming from the content-manager package so I think we should be good.